.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.8
```

Comment:
yeah, you can't do this. You'll need to find a 1.8 version to run gofmt :-) (or do it manually)